articles/native-platforms/ios-objc/05-linking-accounts.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Linking User Profiles
+description: This tutorial will show you how to link different User profiles. Enabling different ways of signing in, into a single profile
+---
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* CocoaPods 1.0.0
+* XCode 7.3 (7D175)
+* Simulator - iPhone 6 - iOS 9.3 (13E230)
+  :::
+
+<%= include('../../_includes/_package', {
+  pkgRepo: 'native-mobile-samples',
+  pkgBranch: 'master',
+  pkgPath: 'iOS/basic-sample-objC',
+  pkgFilePath: 'iOS/basic-sample-objc/ObjCSample/Info.plist',
+  pkgType: 'replace'
+}) %>
+
+### Lets begin
+
+#### Be familiar with Auth0
+
+This tutorial assumes you are already familiar with Auth0 and how to Sign up and Sign in using Lock or Auth0 Toolkit. **If you're not sure, check out [this tutorial](01-login.md) first.**
+
+### 1. Setting up
+
+For this tutorial we are going to use the Auth0 Toolkit, and import the toolkit (that is writen in Swift) into our Objective C project. All the necessary steps for this are covered on the [Custom Login](02-custom-login.md) tutorial. 
+
+### 2. Getting your second token 
+
+To link two user profiles, you need to be signed in with the user you want to use as your primary profile, the one you want to merge the other user into. Once you're signed in, you need to show the login for the second user profile, and for this you have two options.
+
+#### i. Use Lock 
+
+You can use Lock, present the Auth0 sign in UI where the user can select what form of sign in he wants to use.
+
+```objc
+    A0LockViewController *controller = [lock newLockViewController];
+    controller.onAuthenticationBlock = ^(A0UserProfile *profile, A0Token *token) {
+        // dismiss the ViewController
+        [self dismissViewControllerAnimated:YES completion:nil];
+        // Link with the other user profile
+    };
+    
+    [self presentViewController:controller animated:YES completion:nil];
+
+```
+
+#### ii. Use Auth0 Toolkit
+
+You might chose to avoid the Lock UI and only show the sign in options of 3rd parties, for example. In this case you'd be encouraged to read the [Custom Login](02-custom-login.md) tutorial. But here we'll have a quick outline.
```

Comment:
You might choose to avoid the Lock UI and only show the sign in options from third parties. If that's the case, you're encouraged to read the [Custom Login](02-custom-login.md) tutorial. Anyway, here you have a quick outline.